.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # CODEOWNERS: https://help.github.com/articles/about-codeowners/
 
 # Primary repo maintainers
-*       @alexanderbez @alessio @fedekunze @nylira @hschoenburg @aaronc @clevinson @jgimeno
+*       @alexanderbez @alessio @fedekunze @nylira @aaronc @clevinson @jgimeno
```

Comment:
I don't want to sound rude, but is there a reason for @nylira to be here? He doesn't contribute code, create issues nor participates in technical SDK discussions...

Reply:
I don't have to be on this list. I trust the other CODEOWNERS to make good engineering decisions for the Cosmos ecosystem.